Resources/Prototypes/DeltaV/Entities/Objects/Specific/Mail/mail.yml

```diff
@@ -1732,12 +1790,3 @@
       amount: 1
     - id: PaperMailHydroCoTrial
       amount: 1
-
-- type: entity
```

Comment:
undo this removal

Reply:
WHOOPS!

Reply:
I'll do it, but I did mention it's also in the sword mail which is higher up in the file now. So like... ???